apps/server/src/shared/domain/rules/task.rule.spec.ts

```diff
@@ -145,5 +147,20 @@ describe('TaskRule', () => {
 			const res = service.hasPermission(student, entity, { action: Actions.read, requiredPermissions: [permissionC] });
 			expect(res).toBe(false);
 		});
+
+		it('should return "false" if task has Assignees and user is not assigned ', () => {
+			const otherStudent: User = userFactory.build({ roles: [role] });
```

Comment:
Maybe it make sense to increase the factory. 
https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/shared/testing/factory/user.factory.ts#L8

withRoleByName(name: RoleName) get a optional command to pass permissions. Or withRole can get permissions instead of a role.

userFactory.withRole([permissions...]).build();